frameworks/Dart/aqueduct/aqueduct.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM google/dart:1.24
+
+COPY ./ ./
+
+RUN pub upgrade
```

Comment:
I see that our other Dart tests have this `pub upgrade` line as well, but I'm curious of your opinion - Is this something we should be doing?  Is it possible for this to upgrade a dependency such that it pulls in breaking changes?

Reply:
As long as the Developer properly constains their pubspec.yaml then pub upgrade should only apply bug fix versions and not breaking changes.

A fresh pub get would do virtually the same thing without a pub.lock existing.

Reply:
Got it, thanks.